src/main/webapp/script.js

```diff
@@ -207,3 +210,35 @@ for (let button of filterButtons) {
       }
     });
 }
+
+let infowindow;
+function updateRestaurantInfo(restaurant, marker) {
+  if (!infowindow) {
+    infowindow = new google.maps.InfoWindow();
+  }
+  let { geometry: {location}, placeId, name, photos, rating } = restaurant;
+
+  let params = {
+    api: 1,
+    destination: name,
+    destination_place_id: placeId
+  }
+  const dir_url = new URL('https://www.google.com/maps/dir/');
+  Object.keys(params).forEach(key => dir_url.searchParams.append(key, params[key]));
+
+  let contentString =
```

Comment:
For readability maybe leaving it as a single string and using template literals would be better?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals